### PR TITLE
Change convention from BGR to RGB

### PR DESCRIPTION
**Pull request by yuyu2172**
Fix points raised in #187.
Weights uploaded on the internet and the links to them will be changed in a different PR.

### CODE REVIEW

**Comment by Hakuyume** [comment]
Could you fix `trainsfroms.pca_lighting`?

**Comment by yuyu2172** [comment]
Right. Thanks for pointing that out.

File: chainercv/transforms/image/pca_lighting.py [hunk 1]
```diff
@@ -38,9 +38,9 @@ def pca_lighting(img, sigma, eigen_value=None, eigen_vector=None):
         eigen_value = np.array((0.2175, 0.0188, 0.0045))
     if eigen_vector is None:
         eigen_vector = np.array((
-            (0.4009, -0.814,  0.4203),
+            (-0.5675, -0.5808, -0.5836),
```
Comment (Hakuyume) [inline chainercv/transforms/image/pca_lighting.py]: Could you add "img is CHW and RGB format" to `Args` section?

Comment (yuyu2172) [inline chainercv/transforms/image/pca_lighting.py]: Ok

**Comment by Hakuyume** [comment]
LGTM

**Pull request merged**